src/Microsoft.Framework.PackageManager/NuGet/Core/Configuration/Settings.cs

```diff
@@ -601,6 +601,24 @@ public bool DeleteSection(string section)
             return true;
         }
 
+        public IEnumerable<string> GetNuGetConfigFiles()
+        {
+            var next = _next;
```

Comment:
Realized I start on the second element in the list here, will change to start at the beginning.